src/utilities/email-helper.ts

```diff
@@ -44,7 +50,22 @@ class EmailHelper {
         };
 
         this.client = nodemailer.createTransport(sgTransport(clientOptions));
-
+        this.mailer = new Email({
+            message: {
+                from: this.from,
+                // Default attachments for all emails without the email client automatically blocking them.
+                // GMail and Outlook do not support base64 data-urls.
+                attachments: [
+                    {
+                      filename: 'favicon.png',
+                      path: 'emails/rederly_favicon.png',
+                      cid: 'favicon'
+                    }
+                ]
+            },
+            transport: this.client,
+            send: true,
```

Comment:
What is this flag for?
Should it be sending whether or not email is disabled?

Reply:
I see, the original logic still stands, still what is the purpose of this flag?

Reply:
Good question; this flag is to force sending when in development mode. If it were false, then only the local preview would be generated. This shouldn't matter since our configuration for emails are handled at the start of the function.